### PR TITLE
Remove env file requirement

### PR DESCRIPTION
**Pull request by hexyls**
Fixes https://github.com/protofire/omen-exchange/issues/1152

### CODE REVIEW

**Comment by liaojikunwork** [comment]
@hexyls , could you resolve conflicts please?

File: app/src/index.tsx [hunk 1]
```diff
@@ -2,11 +2,5 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 
 import App from './App'
-import * as serviceWorker from './serviceWorker'
```
Comment (liaojikunwork) [inline app/src/index.tsx:5]: @hexyls 
Why did you remove serviceWorker?

Comment (hexyls) [reply app/src/index.tsx:5]: haha oh yeah I noticed we aren't using it (we unregister the service worker immediately [here](https://github.com/protofire/omen-exchange/pull/1215/files/0f51a2513eb17744690a0ee08924687306a189fc#diff-f2e29ede120f9b093448582666e7c0eab8fd03e8e69f7522e5011901c54819ddL12)) so we are loading that extra code for no reason.

Comment (liaojikunwork) [reply app/src/index.tsx:5]: @hexyls , don't remove serviceWorker. It can be necessary later.
And this issue(PR) is not related to serviceWorker.

**Comment by hexyls** [comment]
There does not appear to be consensus on this yet in https://github.com/protofire/omen-exchange/issues/1152, converted to draft for now.

**Comment by liaojikunwork** [comment]
> There does not appear to be consensus on this yet in #1152, converted to draft for now.

It's still not correct now. Pls check @hexyls

**Comment by hexyls** [comment]
lol yes I know dude, will update now but please don't merge yet.

**Review by liaojikunwork — APPROVED** [review]
